core/src/main/java/com/scalar/db/storage/jdbc/JdbcAdmin.java

```diff
@@ -565,6 +572,8 @@ public void dropNamespace(String namespace) throws ExecutionException {
 
     try (Connection connection = dataSource.getConnection()) {
       execute(connection, dropStatement);
+      deleteFromNamespaceTable(connection, namespace);
+      deleteNamespaceTableIfEmpty(connection);
```

Comment:
When dropping a namespace, the namespace is also deleted from the `namespace` table.